util/env/env.go

```diff
@@ -32,3 +35,26 @@ func ParseNumFromEnv(env string, defaultValue, min, max int) int {
 	}
 	return num
 }
+
+func ParseDurationFromEnv(env string, defaultValue, min, max time.Duration) time.Duration {
```

Comment:
Can we have some commentary and unit tests for this, please?

Reply:
done :)